server-code/Lookups.js

```diff
@@ -0,0 +1,18 @@
+/*
+Document: JS code specific for Lookups
+*/
+
+/**
+ * Produces HTML display output for given WBS #'s project or work package.
+ * 
+ * @param {String} wbsNum – The Work Breakdown Structure # to get project or work package display output for
+ * @return {String} – Raw HTML display output (specifically a description list) corresponding to the project or 
+ *                    work package attached to the specified work breakdown structure #                   
+ */
+
+function getProjectOrWorkPackage(wbsNum) {
```

Comment:
Do you think there is a better name for this?

Reply:
it's probably fine the way it is, but if you want, you could call it "WBS Element"

Reply:
Resolved by commit 070a9a3ae6cf6a0d6506a18025ddc43b90cd4961